libraries/AP_Proximity/AP_Proximity_RPLidarA2.cpp

```diff
@@ -33,27 +33,32 @@
 #include <stdio.h>
 #include <GCS_MAVLink/GCS.h>
 
-#define RP_DEBUG_LEVEL 1
```

Comment:
The debug_level approach allows one to easily control the level of debug messages verbosity and is more flexible than the simple binary method you added here.

Reply:
A matter of approach I guess, for me it wasn't useful at all. Can revert it if you wish...

Reply:
OK, I updated the macros to include both options (verbosity and destination settings both global and for each instance). Also restructured them to look better.